datafusion/core/tests/parquet/filter_pushdown.rs

```diff
@@ -632,7 +632,7 @@ async fn predicate_cache_pushdown_default() -> datafusion_common::Result<()> {
 #[tokio::test]
 async fn predicate_cache_pushdown_disable() -> datafusion_common::Result<()> {
     // Can disable the cache even with filter pushdown by setting the size to 0. In this case we
-    // expect the inner records are reported but no records are read from the cache
```

Comment:
nit: wording a bit off here, since it reads as

> Can disable the cache even with filter pushdown by setting the size to 0. In this case we no records are read from the cache and no metrics are reported

Should be this maybe?

> Can disable the cache even with filter pushdown by setting the size to 0. This results in no records being read from the cache and no metrics being reported